__tests__/endpoints.test.js

```diff
@@ -77,6 +77,34 @@ describe("/api/articles/:article_id", () => {
         expect(msg).toBe("Article does not exist");
       });
   });
+  test("PATCH 200 - Responds with an article that has a correctly updated vote count", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 50 })
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body).toHaveProperty("votes", 150);
+      });
+  });
+  test("PATCH 400 - Empty increment votes object received", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({})
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+  test("PATCH 400 - Failing schema validation", () => {
```

Comment:
what if the path is /api/articles/1000 

or 

/api/articles/hello

lets add a couple of error tests for this scenario